config/config.yaml

```diff
@@ -249,26 +249,6 @@ tide:
     - do-not-merge/release-note-label-needed
     - do-not-merge/work-in-progress
     - needs-rebase
-  # Maintain separate cert-manager configuration to make it easy to enable code freeze
```

Comment:
👀 sorry I think we've got mixed up - this block *should* still exist, but you need to just remove the `excludedBranches` section from it as we have the same merge requirements for all branches in the repo when we aren't in code freeze